samples/bluetooth/direction_finding_connectionless_rx/README.rst

```diff
@@ -112,6 +112,43 @@ The following table presents the patterns that you can use to switch antennas on
 | RFU    | 15 (0b1111)  |
 +--------+--------------+
 
+Constant Tone Extension transmit and receive parameters
+=======================================================
+
+Constant Tone Extension works in one of two modes: angle of arrival (AoA) or angle of departure (AoD).
+The transmitter is configured with operation mode that is used for CTE transmission.
+The receiver operating mode depends on configuration, by default both AoA and AoD modes are enabled, and CTE type received in CTEInfo filed in PDUs extended advertising header.
+Depending on operation mode selected for reception of CTE, the receivers radio peripheral will execute antenna switching and CTE sampling (AoA) or CTE sampling only (AoA).
+
+There are two antenna switch slot length allowed: 1 us or 2 us.
+The transmitter uses the antenna switch slot length to configure radio peripheral in Angle of Deprature mode only.
+The receiver uses the local setting of antenna switch slot length in Angle of Arrival mode only.
+When CTE type in received PDU is AoD, the receivers radio peripheral uses the antenna switch slot length appropriate for AoD type: 1 us or 2 us.
+The antenna switch slot length has influece on number of IQ samples provided in IQ samples report.
+
+Constant Tone Extension length is limited to range 16 us to 160 us, the value is provided in 8 us units.
+The transmitter is responsible for setting of CTE length.
+The value is send to receiver as part of CTEInfo field in PDUs extended advertising header.
+The receivers radio peripheral uses the CTE length, provided in periodic advertising PDU, to execute antenna switching and CTE sampling in AoA mode or CTE sampling only in AoD mode.
+The CTE length has influence on number of IQ samples provided in IQ samples report.
+
+Constant Tone Extension consists of three periods:
+
+* Guard period that lasts for 4 us. It is a gap before actual CTE reception and adjacent PDU transmission to avoid interference.
```

Comment:
```suggestion
* Guard period that lasts for 4 µs. There is a gap before the actual CTE reception and the adjacent PDU transmission to avoid interference.
```